app/controllers/repairs_controller.rb

```diff
@@ -2,86 +2,46 @@ class RepairsController < ApplicationController
   before_filter :authenticate_user!
   load_and_authorize_resource
 
-  # GET /repairs
-  # GET /repairs.json
   def index
     @repairs = Repair.all
-
-    respond_to do |format|
-      format.html # index.html.erb
-      format.json { render json: @repairs }
-    end
   end
 
-  # GET /repairs/1
-  # GET /repairs/1.json
   def show
-    @repair = Repair.find(params[:id])
-
-    respond_to do |format|
-      format.html # show.html.erb
-      format.json { render json: @repair }
-    end
+    @last_editor = last_editor(@repair)
```

Comment:
side note. i find it extremely weird / wrong that `last_editor` doesn't return a user. Violates rule of least surprise and totally limits its utility. Now, maybe a `last_editor` method and a `last_editor_name` method would be a good combo, the 2nd building on the output of the 1st.

obviously that's outside of the scope of this PR but since it was being used here i went and investigated.